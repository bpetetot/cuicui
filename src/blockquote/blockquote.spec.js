/* eslint-disable react/jsx-filename-extension */
/* eslint-env jest */

import snap from 'snap'

import Blockquote from './blockquote'

const snapshot = props => snap(Blockquote)({ ...props })

describe('blockquote', () => {
  it('should add custom className', snapshot({ className: 'custom' }))
})
