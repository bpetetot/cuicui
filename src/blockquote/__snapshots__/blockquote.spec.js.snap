// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blockquote should add custom className 1`] = `
<blockquote
  className="cc-blockquote custom"
  style={undefined}
>
  <p>
    
  </p>
</blockquote>
`;

exports[`blockquote should add custom style 1`] = `
<blockquote
  className="cc-blockquote"
  style={
    Object {
      "color": "red",
    }
  }
>
  <p>
    
  </p>
</blockquote>
`;

exports[`blockquote should add text 1`] = `
<blockquote
  className="cc-blockquote"
  style={undefined}
>
  <p>
    hello world
  </p>
</blockquote>
`;
